internal/ark/sync/backups_sync_svc.go

```diff
@@ -153,13 +154,21 @@ func (s *BackupsSyncService) SyncBackupsForCluster(cluster api.Cluster) error {
 			log.WithField("count", req.NodeCount).Debug("node count found")
 		}
 
-		_, err = s.backupsSvc.Persist(req)
+		syncedBackup, err := s.backupsSvc.Persist(req)
 		if err != nil {
 			return errors.WrapIf(err, "could not persist backup")
 		}
+		backupIDS = append(backupIDS, int(syncedBackup.ID))
 
 		log.Debug("backup synced")
 	}
 
+	log := s.logger.WithField("bucket", bucket.Name).WithField("clusterId", bucket.ClusterID)
+	log.Debug("removing backups not found on cluster")
```

Comment:
**Question**: shouldn't we also log the `backupIDs` content here for debugging purposes? Is it not helpful in any way?

Reply:
Would be more interesting to see it there were any deleted items, but we don't return here anything like that.